packages/amplify_datastore/example/lib/event_display_widgets.dart

```diff
@@ -1,55 +1,70 @@
-part of sample_app;

```

Comment:
Not sure what this was about - this is the result of adding the license header and running `flutter format`.